### PR TITLE
fix(config/clusters): added policies for S3 access update-driver-website workflow

### PR DESCRIPTION
**Pull request by cappellinsamuele**
This PR adds the policy to allow update-driver-website workflow to upload to S3

### CODE REVIEW

File: config/clusters/iam.tf [hunk 1]
```diff
@@ -218,6 +218,44 @@ data "aws_iam_policy_document" "rules_s3_access" {
...
+    effect = "Allow"
+    actions = [
+      "s3:PutObject",
+      "s3:GetObjectAcl",
+      "s3:GetObject",
+      "s3:DeleteObject",
+      "s3:PutObjectAcl"
+    ]
+    resources = [
+      "arn:aws:s3:::falco-distribution/driver/site/*",
```
Comment (FedeDP) [inline config/clusters/iam.tf:253]: We cannot escape `falco-distribution/driver/site` ;) Thus we should not be able to make any damage (eg: deleting drivers)!

File: config/clusters/iam.tf [hunk 2]
```diff
@@ -218,6 +218,44 @@ data "aws_iam_policy_document" "rules_s3_access" {
   }
 }
 
+# Test-infra repository
+
+module "test-infra_s3_role" {
+  source    = "terraform-aws-modules/iam/aws//modules/iam-github-oidc-role"
+  version = "5.10.0"
```
Comment (LucaGuerra) [inline config/clusters/iam.tf:225]:
```suggestion
  version = "5.10.0"
  name = "github_actions-test-infra-s3"
```
Just a nit but if you add this the role name will be predictable!

**Review by FedeDP — APPROVED** [review]
/approve

**Comment by LucaGuerra** [comment]
Great job @cappellinsamuele ! :rocket:

File: config/clusters/iam.tf [hunk 3]
```diff
@@ -218,6 +218,45 @@ data "aws_iam_policy_document" "rules_s3_access" {
...
+  name = "github_actions-test-infra-s3"
+  create = true
+  subjects = [
+    "falcosecurity/test-infra:ref:refs/heads/master"
```
[the remark this answers is not in the document]
Comment (FedeDP) [reply config/clusters/iam.tf:229]: :pray:

**Pull request merged**